learn/getting_started/algolia_migration.md

```diff
@@ -0,0 +1,249 @@
+3. (optional) Configure your settings to approximate the settings of your Algolia index
+
+This guide uses a [Node.js](https://nodejs.org/en/) script to upload Algolia index data to Meilisearch. If you would like to write this script in another language, we provide [SDKs for most common languages](/learn/what_is_meilisearch/sdks.md) and [welcome contributions to this guide](https://github.com/meilisearch/documentation/edit/master/learn/getting_started/algolia_migration.md).
+
+Before continuing, make sure you have both Meilisearch and Node.js installed, and access to a command-line terminal. If you're unsure, see our [quick start](/learn/getting_started/quick_start.md) for instructions on installing Meilisearch.
+
+### Initialize project
+
+Start by creating a directory `algolia-meilisearch-migration` and initializing it as an npm package:
+
+```bash
+mkdir algolia-meilisearch-migration
+cd algolia-meilisearch-migration
+npm init -y
+```
+
+This will set up the environment we need to install dependencies.
+
+Next, create a `script.js` file:
+
+```bash
+touch script.js
+```
+
+This file will contain our migration script.
+
+### Install required packages
+
+To get started, you'll need two different packages. The first is `algoliasearch`, the JavaScript client for the Algolia API, and the second is `meilisearch`, the JavaScript client for the Meilisearch API.
+
+```bash
+npm install algoliasearch meilisearch
+```
+
+### Create Algolia client
+
+You'll need your **Application ID** and **API Key** to start the Algolia client. Both can be found in your [Algolia account](https://www.algolia.com/api-keys).
+
+Paste the below code in `script.js`:
+
+```js
+const AlgoliaSearch = require("algoliasearch");
+const algoliaClient = AlgoliaSearch(
+  "APPLICATION_ID",
+  "ADMIN_API_KEY"
+);
+const algoliaIndex = algoliaClient.initIndex("INDEX_NAME");
+```
+
+Replace `APPLICATION_ID` and `ADMIN_API_KEY` with your Algolia application ID and admin API key respectively.
+
+Replace `INDEX_NAME` with the name of the Algolia index you would like to migrate to Meilisearch.
+
+### Fetch data from Algolia
+
+To fetch all Algolia index data at once, use Algolia's [`browseObjects`](https://www.algolia.com/doc/api-reference/api-methods/browse/) method.
+
+```js
+let records = [];
+await algoliaIndex.browseObjects({
+    batch: (hits) => {
+      records=records.concat(hits)
+    }
+  });
+```
+
+On each batch of hits, the `batch` callback method is invoked, and the content is concatenated in the `records` array. This variable will be used later on in the upload process.
+
+#### Nested objects
+
+Meilisearch versions prior to v0.27.0 have limited support for nested fields, meaning that **nested objects must be flattened manually**. This can be avoided entirely by using [the latest version of Meilisearch](https://github.com/meilisearch/MeiliSearch/releases). If this is impossible for some reason, try the modified data retrieval process below.
```

Comment:
I prefer to leave patch versioning outside of versions unless the patch is relevant to the subject.

```suggestion
Meilisearch versions prior to v0.27 have limited support for nested fields, meaning that **nested objects must be flattened manually**. This can be avoided entirely by using [the latest version of Meilisearch](https://github.com/meilisearch/MeiliSearch/releases). If this is impossible for some reason, try the modified data retrieval process below.
```